crabs/detection_tracking/train_model.py

```diff
@@ -99,7 +102,7 @@ def setup_trainer(self):
             accelerator=self.accelerator,
             logger=mlf_logger,
             enable_checkpointing=enable_checkpointing,
-            callbacks=checkpoint_callback,
+            callbacks=[checkpoint_callback] if checkpoint_callback else [],
```

Comment:
Question: I think we don't need to pass a list here right?
```suggestion
            callbacks= checkpoint_callback,
```